components/layout/display_list_builder.rs

```diff
@@ -1834,6 +1841,8 @@ impl FragmentDisplayListBuilding for Fragment {
         let stacking_relative_content_box =
             self.stacking_relative_content_box(stacking_relative_border_box);
 
+        let clipping_region = clip_for_border_radius(&self.style, &stacking_relative_content_box);
```

Comment:
I tried ```clip``` and it didn't work, but strangely it works with ``stacking_relative_content_box``` so I just used that. I don't really have any oversight over this code so I don't know why.